examples/connection_snippets.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(dead_code)]
+#![allow(unused_imports)]
+#![allow(unused_variables)]
+
+extern crate mongodb;
+
+use mongodb::error::Result;
+
+const CONNECTION_URI: &str =
```

Comment:
since the Atlas team will just be copy/pasting this example, I think we actually need to have this string literal in the examples themselves, though I do agree that in "real" code we should use a constant.